integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -133,11 +138,19 @@ def transform_revision_number(self, content_metadata_item):  # pylint: disable=u
         """
         return 1
 
-    def transform_schedule(self, content_metadata_item):  # pylint: disable=unused-argument
+    def transform_schedule(self, content_metadata_item):
         """
         Return the schedule of the content item.
         """
-        return []
```

Comment:
did we just assume we couldn't export schedule before? Wish we had a comment explaining the empty array return value

Reply:
yeah I can't find anything like that. The SAP docs as of today do say they prefer schedule if given, otherwise, that 'duration' is used. Since we are only sending schedule anyway I did not worry about duration fwiw.

Reply:
in this case I figured it would be better to always send the same shape in this field regardless of if we found a course run or not. I am presuming that is better than not sending anything at all (like rn)

Reply:
it is honestly a bit hard to know what the best answer if for SAP since we don't hav a test environment like the others, so fo this one we will have to transmit, then maybe get @mattdrayer 's help in seeing what it looks like on customers' side